<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel">
     <property name="text">
      <string>Configuration:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="configComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="editConfigButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>37</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel">
     <property name="text">
      <string>Mode:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="modeXButton">
     <property name="text">
      <string>x</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttongroup_xy</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="modeYButton">
     <property name="text">
      <string>y</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttongroup_xy</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="modeXYButton">
     <property name="text">
      <string>x/y</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttongroup_xy</string>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttongroup_xy"/>
 </buttongroups>
</ui>
